Main.java

```diff
@@ -1,15 +1,34 @@
+import java.util.Scanner;
 /**
- *
- * @author 
+ * which of the 2 numbers input is the largest one.
+ * @author Ahmad Zeshan 
  */
 public class Main {
 
   /**
    * The method that is executed when you hit the run button.
    * @param args the command line arguments
    */
-  public static void main(String[] args) {
-    // TODO code application logic here
+  public static void main(String[] args) {
+		// create a Scanner for user input
+		Scanner input = new Scanner(System.in);
+		//prompts user to enter a integer input
+		System.out.println("Please enter an integer:");
+		//gets input from user
+		int x = input.nextInt();
+		//prompts user for another integer
+		System.out.println("Please enter another integer:");
+		//gets the input from user
+		int x1 = input.nextInt();
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. x and x1 are poor variable names here.